galpy/potential/amuse.py

```diff
@@ -195,9 +205,10 @@ def enclosed_mass(self,r):
         HISTORY:
            2019-08-12 - Written - Webb (UofT)
         """
-        vc2= potential.vcirc(self.pot,r.value_in(units.kpc)/self.ro,phi=0,
-                             t=self.tgalpy,ro=self.ro,vo=self.vo)**2.
-        return vc2*r.value_in(units.kpc)/bovy_conversion._G*1000. | units.MSun
+        vc2= potential.vcirc(self.pot,r.value_in(u.kpc)/self.ro,phi=0,
+                            t=self.tgalpy,ro=self.ro,vo=self.vo,
+                            use_physical=False)**2.
+        return vc2*r.value_in(u.kpc)/bovy_conversion._G*1000.|u.MSun
```

Comment:
Should probably check the units here, not sure this is correct from looking at it (probably not)

Reply:
All it needs is to change `vc2= potential.vcirc(self.pot,r.value_in(u.kpc)/self.ro,phi=0,
                            t=self.tgalpy,ro=self.ro,vo=self.vo,
                            use_physical=False)**2.` to `self.circular_velocity(r)**2.` Then the corrections to those units sort everyone else out.

Reply:
I've implemented this in a new commit.